src/D_EcobeeHouse1.json

```diff
@@ -35,10 +35,10 @@
     ],
     "inScene": "1",
     "DisplayStatus": {
-        "Service": "urn:upnp-org:serviceId:SwitchPower1",
-        "Variable": "Status",
-        "MinValue": "0",
-        "MaxValue": "1"
+        "Service": "urn:ecobee-com:serviceId:Ecobee1",
+        "Variable": "currentClimateRef",
+        "MinValue": "home",
+        "MaxValue": "smart1"
```

Comment:
Looking further into it. It looks like the ecobee vacation concept gets created through a schedule and the mode actually doesn't exist until it is scheduled? If I am not mistaken, one would have to set and send the parameters every time from the vera? Seems more convenient to have a preset comfort setting for it.